### ISSUE TITLE
maybe --start-address and --stop-address dont work when i attach --macho or -m options on commandline

### ISSUE THREAD

**Issue by kimyw3007** [llvm/llvm-project#138]
Hi i found out some wired behavior

maybe i don't understand exactly how this works

anyway, i faced some issues when i used this

if i attach "--start-address and --stop-address" options don't work with --macho or -m options on commandline
so if i try to this options(--start-address and --stop-address) with -macho or -m option 

this job fall in infinite loop when end of binary

Didn't i understand anything of this command?

please check this 

Thank you for reading

**Comment by androm3da** [comment]
Does it only behave that way with some files or any one you've tried does that?

**Comment by kimyw3007** [comment]
> Does it only behave that way with some files or any one you've tried does that?

nope, it doesn't work with all archive files (in {name}.ipa/{name}.app/{name} binary) with these options and i searched MachOOpt option in this source code and this branch and In MachOOpt i didn't find  references of start-address and stop-address (maybe i still not find yet)
